Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridView.cs

```diff
@@ -225,6 +225,16 @@ protected override void Dispose(bool disposing)
     #endregion
 
     #region Public New
+    /// <inheritdoc/>
+    [Category(@"Behavior")]
+    [DefaultValue(true)]
+    [DesignerSerializationVisibility(DesignerSerializationVisibility.Visible)]
+    public new bool DoubleBuffered 
+    {
+        get => base.DoubleBuffered;
+        set => base.DoubleBuffered = value;
```

Comment:
How about triggering Invalidate on toggle?
```suggestion
    set 
    { 
        if (base.DoubleBuffered != value)
        {
            base.DoubleBuffered = value;
            this.Invalidate();
        }
    }
```

Reply:
Updated.